components/ModDashboardHealthProfessionalCard.vue

```diff
@@ -51,7 +51,10 @@
                     </div>
                 </div>
             </div>
-            <div v-if="moderationScreenStore.activeScreen === ModerationScreen.EditSubmission">
+            <div
+                v-if="moderationScreenStore.activeScreen === ModerationScreen.EditSubmission
```

Comment:
a good design practice is for components to never know where they live on a page. (this encourages reusability)
let's move this v-if so the parent can pass in this condition as `isEditable`

Reply:
I know @ermish that components shouldn't know where they are as a design. But this has nothing to do with if it is editable. It just is whether it displays the names as a locale because that is what you change for a healthcare professional or whether it shows other information when in the `edit-facility` page based on the pertinent information to the user. 

Is there a more standard way there? Like giving it a type as a prop? Like `healthcarepProfessionalView` and `facilityView`

Reply:
Or maybe to be clearer. Not specifically if it is editable. Since they can be used to delete HPs in the facility as well